modules/ROOT/pages/branding-and-customization.adoc

```diff
@@ -4,98 +4,103 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Use the Community Builder to customize your community and add branding, images and logos.
+Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
 
-In the ACM administrator interface, select the *Community Builder* link.
+Select *Community Builder* in the ACM administrator interface. By default, 
+the Community Builder displays the *Home* page. You can select the *Home* drop-down list arrow to display a list 
```

Comment:
This is the *Pages* drop-down list, and always displays the title of the current page, which is sometimes *Home* but not always.